fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/internal/ClouddriverService.java

```diff
@@ -25,4 +26,7 @@
 
   @GET("/credentials")
   List<Account> getAccounts();
+
+  @GET("/applications?restricted=false&expand=false")
```

Comment:
I think you can use arguments and annotations to avoid hardcoding this into the endpoint. Maybe something like https://github.com/spinnaker/clouddriver/blob/master/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/services/Front50Service.groovy#L41?